<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveAccountPage</class>
 <widget class="QDialog" name="RemoveAccountPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RemoveAccountPage{
background-color: #eaf2f8;
}</string>
  </property>
  <widget class="QPushButton" name="removeAccountButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> #removeAccountButton {
        background-color: #2980b9;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px 16px;
        font-size: 14px;
    }
    #removeAccountButton:hover {
        background-color: #3498db;
    }
    #removeAccountButton:pressed {
        background-color: #1c5980;
    }</string>
   </property>
   <property name="text">
    <string>حذف حساب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>لطفا شماره حساب مدنظر را وارد کنید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountNumberField">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#accountNumberField {
        border: 1px solid #2c3e50;
        border-radius: 6px;
        padding: 6px;
        background-color: #ecf0f1;
        font-size: 14px;
	    color: rgb(0, 0, 0);
 }
 #accountNumberField:focus {
        border: 1px solid #2980b9;
        background-color: #ffffff;
    }</string>
   </property>
  </widget>
  <widget class="QListWidget" name="accountNumberList">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#accountNumberList {
	color: rgb(0, 0, 0);
        background-color: #f8f9fa;
        border: 1px solid #bdc3c7;
        font-size: 14px;
        padding: 4px;
    }
    #accountNumberList::item {
        padding: 8px;
        border-bottom: 1px solid #dcdcdc;
    }
    #accountNumberList::item:selected {
        background-color: #d6eaf8;
        color: #154360;
    }</string>
   </property>
  </widget>
  <widget class="QListWidget" name="accountTypeList">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#accountTypeList {
	color: rgb(0, 0, 0);
        background-color: #f8f9fa;
        border: 1px solid #bdc3c7;
        font-size: 14px;
        padding: 4px;
    }
    #accountTypeList::item {
        padding: 8px;
        border-bottom: 1px solid #dcdcdc;
    }
    #accountTypeList::item:selected {
        background-color: #d6eaf8;
        color: #154360;
    }</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ownerList">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#ownerList {
	color: rgb(0, 0, 0);
        background-color: #f8f9fa;
        border: 1px solid #bdc3c7;
        font-size: 14px;
        padding: 4px;
    }
    #ownerList::item {
        padding: 8px;
        border-bottom: 1px solid #dcdcdc;
    }
    #ownerList::item:selected {
        background-color: #d6eaf8;
        color: #154360;
    }</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
